content/tools/sdk/JenkinsPlugin.md

```diff
@@ -6,11 +6,9 @@ tags: "tools Jenkins SDK Integration Java"
 
 [Jenkins](https://jenkins.io/) is a continuous integration (CI) application that can be used to build, deploy and run applications. The Contrast Jenkins Plugin is a tool for integrating Contrast with your Jenkins CI instance. You can use it to test your connection to Contrast and verify your build with threshold conditions.
 
-## Use The Plugin
+## Use the Plugin
 
-You can view the plugin code in Contrast's [Github repository](https://github.com/Contrast-Security-OSS/contrast-jenkins-plugin). In the Jenkins dashboard, go to **Manage Jenkins** in the left sidebar and select the **Configure System** page to find a new Contrast TeamServer profiles section.
-
-<!-- The plugin can be found here on the Jenkins repository. -->
+You can view the plugin code in Jenkins' [Github repository](https://github.com/jenkinsci/contrast-continuous-application-security-plugin). In the Jenkins dashboard, go to **Manage Jenkins** in the left sidebar, and select the **Configure System** page to find a new Contrast TeamServer profiles section.
```

Comment:
Is "TeamServer" referenced on the page by name?

Reply:
Yes it is. The section is titled "Contrast Teamserver profiles" and has a few fields under it. None of the fields say teamserver though. Even the URL field just says "Contrast URL". 

![image](https://user-images.githubusercontent.com/14810035/34958045-fe45e1c6-f9e4-11e7-8157-d3764c590a78.png)